test/verify/check-machines-dbus

```diff
@@ -277,7 +277,7 @@ class TestMachinesDBus(machineslib.TestMachines):
         m.execute("virsh attach-interface --persistent VmNotInstalled bridge virbr0")
 
         # Change the os boot firmware configuration
-        supports_firmware_config = m.image in ['fedora-31', 'fedora-testing', 'debian-testing', 'ubuntu-stable']
+        supports_firmware_config = m.image in ['fedora-31', 'fedora-testing', 'debian-testing', 'ubuntu-stable'] and m.execute("virsh --version") >= "5.2.0"
```

Comment:
so what is the point of having list of images when you check then for the version anyway?
And please note, that comparing versions as string can backfire very easily.

Reply:
And how does fedora 31 have newer version that fedora testing? That does not make any sense.

Reply:
Right, the point of such static OS lists is to notify us when things change, so that we can adjust them and check against whether the expectations match reality. Otherwise they could just silently be skipped again.